src/common.cpp

```diff
@@ -197,14 +197,24 @@ update_test(const char *dst_path,
 
 	struct stat src_st;
 	stat(src_path, &src_st);
-
-	if (src_st.st_mtim.tv_sec > dst_st.st_mtim.tv_sec) {
+    
+#if (defined __APPLE__)
+	if (src_st.st_mtime > dst_st.st_mtime) {
 		return true;
 	}
 
-	if (src_st.st_mtim.tv_nsec > dst_st.st_mtim.tv_nsec) {
+	if (src_st.st_mtime > dst_st.st_mtime) {
```

Comment:
On macOS, st_mtime is defined as st_mtimespec.tv_sec, which is not the original meaning of author. 
Please see #33 for workaround.